api/foreign.go

```diff
@@ -679,6 +856,32 @@ func (entity *ForeignBusinessConferences) Valid() (bool, error) {
 	return entity.List.Valid()
 }
 
+// ClearNos clears the "no" answers on application rejection
+func (entity *ForeignBusinessConferences) ClearNos() error {
```

Comment:
this whole method, bar the fist line, is nearly identical to its counterpart in ForeignActivitiesDirect.

I wonder if it might be possible to abstract this logic out & to have both call into the common logic with string value for the event Items collection & maybe a custom error string to output?